six/modules/c++/six/include/six/ReadControl.h

```diff
@@ -89,19 +89,17 @@ class ReadControl
                       const std::vector<std::string>& schemaPaths) = 0;
 
     /*!
-     *  Get a const pointer to the current container.  This
-     *  container is not allocated, nor deleted by this class
+     *  Get a const pointer to the current container.
```

Comment:
This and other comments should be updated to reflect a shared pointer is returned